style.css

```
@@ -0,0 +1,71 @@
+body {
+  background-color: rgb(141, 107, 56);
+}
+
+.classbody {
+  background-color: rgb(211, 162, 82);
+  width: 432px;
+  margin-left: 500px;
```

If you use auto margins, your card will be centered in the browser window:

```suggestion
  margin-left: auto;
  margin-right: auto;
```